extensionsGallery-insider.json

```diff
@@ -4816,6 +4816,67 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+{
+					"extensionId": "94",
```

Comment:
Update total count in this file too

Reply:
updated. note that this has 1 more (80) than in the other file (79).

Reply:
yes, that is the mysql extension.